src/pages/versions/config.json

```diff
@@ -63,6 +63,12 @@
         }
       ],
       "historyData": [
+        {
+          "versionLabel": "2.1.1",
+          "docUrl": "/docs/2.1.1/introduction",
```

Comment:
Change `"docUrl": "/docs/2.1.1/introduction",` to `"docUrl": "/docs/2.1.1/intro/about",`